Schedule.API.IntegrationTests/ExaminationTests.cs

```diff
@@ -71,5 +72,49 @@ public async void Gets_examination_recommendation()
 
             Assert.Contains("\"id\":1", responseString);
         }
+
+        [Fact]
+        public async void Check_rooms_availability_with_overlaps()
+        {
+            var client = _factory.CreateClient();
+            var content = JsonContent.Create(new EquipmentRelocationDto
+            {
+                SourceRoomId = 2,
+                DestinationRoomId = 1,
+                TimeInterval = new TimeInterval
+                {
+                    Start = new DateTime(2022, 3, 3, 9, 0, 0),
+                    End = new DateTime(2022, 3, 3, 10, 30, 0)
+                },
+                Amount = 5,
+                EquipmentType = "Chair"
+            });
+            var response = await client.PostAsync("schedule/examination/check-rooms", content);
+            string responseString = await response.Content.ReadAsStringAsync();
+
+            Assert.Contains("1", responseString);
+        }
+
+        [Fact]
+        public async void Check_rooms_availability_without_overlaps()
+        {
+            var client = _factory.CreateClient();
+            var content = JsonContent.Create(new EquipmentRelocationDto
+            {
+                SourceRoomId = 2,
+                DestinationRoomId = 1,
+                TimeInterval = new TimeInterval
+                {
+                    Start = new DateTime(2022, 11, 11, 9, 0, 0),
+                    End = new DateTime(2022, 11, 11, 11, 30, 0)
+                },
+                Amount = 5,
+                EquipmentType = "Chair"
+            });
+            var response = await client.PostAsync("schedule/examination/check-rooms", content);
+            string responseString = await response.Content.ReadAsStringAsync();
+
+            Assert.True(responseString.Equals("[]"));
+        }
```

Comment:
These two tests are identical. The only difference is in assert.

Reply:
Assert and dates :)

Reply:
Second one is check for case when both rooms are available (then empty list should be returned).